docs/installation/migrate.rst

```diff
@@ -16,77 +16,130 @@ You can set up continuous migration of |sr| to maintain a hybrid deployment ("ex
 cloud") or "lift and shift" all to |ccloud| using a one-time migration.
 
 Continuous Migration
---------------------
+~~~~~~~~~~~~~~~~~~~~
 
 For continuous migration, you can use your self-managed |sr| as a primary and
 |sr-ccloud| as a secondary. New schemas will be registered directly to the
 self-managed |sr|, and |crep| will continuously copy schemas from it to
 |sr-ccloud|, which is set to IMPORT mode.
 
-**TBD diagram showing continuous migration**
-
 One-time Migration
-------------------
+~~~~~~~~~~~~~~~~~~
 
 Choose a one-time migration to move all data to a fully-managed |ccloud|
 service. In this case, you migrate your existing self-managed |sr| to
 |sr-ccloud| as a primary. All new schemas are registered to |sr-ccloud|. In the
 scenario, there is no migration from |sr-ccloud| back to the self-managed |sr|.
 
-**TBD diagram showing one-time migration**
+Prerequisites
+-------------
 
+The Run Book describes how to perform a |sr| migration applicable to any type of
+deployment (from on-premise or data center to |sr-ccloud|). The examples also
+serve as a quick start tutorial you can use to walk through the basics of
+migrating schemas from a local cluster to |ccloud|. 
+
+Before you begin, verify that you have:
+
+* |ccloud| `5.2` or newer to serve as the destination |sr|
+* a local install of |cp| (for example, from a :ref:`ce-quickstart` download), or other cluster to serve as the origin |sr|
+
+Schema migration requires that you configure and run |crep-full|. If you need
+more information than is included in the examples here, refer to the
+:ref:`replicator tutorial` <replicator-quickstart>.
 
---------
 Run Book
 --------
 
 To migrate |sr| to |ccloud|, follow these steps:
 
-#.  Set the destination Schema Registry to IMPORT mode.  For example: 
+#.  Start the origin cluster.
 
+    If you are running a local cluster (for example, from a :ref:`ce-quickstart` download),
+    start only |sr| for the purposes of this tutorial.
+    
     .. code:: bash
+  
+        <path-to-confluent>/bin/confluent schema-registry
+    
+    Verify that `schema-registry`, `kafka`, and `zookeeper` are running and that
+    other services are down, as they can conflict with schema migration.
+    
+    ::
+    
+         control-center is [DOWN]
+         ksql-server is [DOWN]
+         connect is [DOWN]
+         kafka-rest is [DOWN]
+         schema-registry is [UP]
+         kafka is [UP]
+         zookeeper is [UP]
 
-        curl -X PUT -H "Content-Type: application/json" -H "Authorization: Basic dGVuYW50Mi1rZXk6bm9oYXNo" "http://destregistry:8081/mode" --data '{"mode": "IMPORT"}'
+#.  Set the destination |sr| to IMPORT mode.  For example: 
 
-#.  Configure Replicator with Schema Registry information.
+    .. code:: bash
+    
+        curl -u <schema-registry-api-key>:<schema-registry-api-secret> -X PUT -H "Content-Type: application/json" "http://destregistry:8081/mode" --data '{"mode": "IMPORT"}'
```

Comment:
Change `destregistry` for something like `<destination SR>` so the user knows a replacement is required

Reply:
It's going to be https to CCSR, not http

Reply:
Consider advising the user that if subjects exist, setting the mode will fail.  The response will be:

```bash
{"error_code":42205,"message":"Cannot import since found existing subjects"}
```

And that the user should delete those subjects first before proceeding.